services/121-service/src/base-audited.entity.ts

```diff
@@ -5,3 +5,8 @@ export class Base121AuditedEntity extends Base121Entity {
   @Column()
   public userId: number;
 }
+
+export class Base121OptionalAuditedEntity extends Base121Entity {
```

Comment:
I don't understand this change. Maybe it would be a nice idea to add a comment here explaining what is going on and/or why this is needed.